crates/crashtracker/Cargo.toml

```diff
@@ -14,7 +14,13 @@ path = "src/bin/receiver.rs"
 
 [dependencies]
 anyhow = "1"
-datadog-crashtracker = { git = "https://github.com/DataDog/libdatadog.git", tag = "v14.3.1"  }
+datadog-crashtracker = { git = "https://github.com/DataDog/libdatadog.git", tag = "v15.0.0"  }
 napi = { version = "2", features = ["serde-json"] }
 napi-derive = { version = "2", default-features = false }
 rustls = { version = "*", default-features = false, features = ["aws-lc-rs"] }
+
+[profile.release]
```

Comment:
This is already done in the root Cargofile.

Reply:
oops, fixed in [7a8bb90](https://github.com/DataDog/libdatadog-nodejs/pull/44/commits/7a8bb9065afc0acffd3e182a3293590be909c946)